transport/http/constants.go

```diff
@@ -46,6 +46,7 @@ const (
 
 	// Name of the service to which the request is being sent. This
 	// corresponds to the Request.Service attribute.
+	// in responses to check requests get processed by correct service.
```

Comment:
Whoops I think you misunderstood me here. I was just suggesting rephrasing the last line. The whole comment should read as:
```
// Name of the service to which the request is being sent. This
// corresponds to the Request.Service attribute. This header is also used
// in responses to ensure requests are processed by the correct service.
```